data/json/mapgen/house08.json

```diff
@@ -1,124 +1,109 @@
-                { "point": "terrain", "id": "t_tree_young", "x": [ 2, 14 ], "y":[ 17,21 ] , "repeat": [ 3,5 ] }
+            "set" : [
+                { "point" : "terrain", "id" : "t_dirt",       "x" : [ 0, 0], "y" : [ 0,23], "repeat" : [5,10] },
+                { "point" : "terrain", "id" : "t_dirt",       "x" : [23,23], "y" : [ 0,23], "repeat" : [5,10] },
+                { "point" : "terrain", "id" : "t_dirt",       "x" : [ 0,23], "y" : [23,23], "repeat" : [5, 8] },
+                { "point" : "terrain", "id" : "t_dirt",       "x" : [ 0,14], "y" : [ 0, 0], "repeat" : [5, 8] },
+                { "point" : "terrain", "id" : "t_tree_apple", "x" : [ 0,14], "y" : [ 0, 0], "repeat" : [1, 2] },
+                { "point" : "terrain", "id" : "t_tree",       "x" : [ 2,14], "y" : [17,21], "repeat" : [2, 6] },
+                { "point" : "terrain", "id" : "t_tree_young", "x" : [ 2,14], "y" : [17,21], "repeat" : [3, 5] }
             ],
-            "comment": "randomly picked from a group ---v",
-            "place_items": [
-                { "item": "bed", "chance": 80,  "x": [4,5 ], "y": [7,8 ] },
-                { "item": "bedroom", "chance": 70, "x": [ 2,8 ], "y": [ 6,8 ] },
-                { "item": "livingroom", "chance": 80, "x": [ 9,21 ], "y": [ 2,7 ] },
-                { "item": "homeguns", "chance": 20, "x": 2, "y": [ 10,11 ] },
-                { "item": "dresser", "chance": 80, "x": 2, "y": [ 10,11 ] },
-                { "item": "dresser", "chance": 50, "x": [ 2,2 ], "y": [ 7,7 ] },
-                { "item": "fridge", "chance": 50, "x": [ 19,19 ], "y": 11 },
-                { "item": "kitchen", "chance": 50, "x": [ 21,21 ], "y": 11 },
-                { "item": "kitchen", "chance": 50, "x": [ 18,18 ], "y": 11 },
-                { "item": "kitchen", "chance": 70, "x": [ 18,20 ], "y": [ 7,10 ] },
-                { "item": "oven", "chance": 50, "x": [ 21, 21 ], "y": 10 },
-                { "item": "cleaning", "chance": 50, "x": [ 2,5 ], "y": [ 13,14 ] },
-                { "item": "softdrugs", "chance": 70, "x": [ 2,5 ], "y": [ 12,14 ] },
-                { "item": "office", "chance": 50, "x": [ 2,7 ], "y": [ 2,4 ] }
+            "place_items" : [
+                { "item" : "bed",        "chance" : 80, "x" : [ 4, 5], "y" : [ 7, 8] },
+                { "item" : "bedroom",    "chance" : 70, "x" : [ 2, 8], "y" : [ 6, 8] },
+                { "item" : "livingroom", "chance" : 80, "x" : [ 9,21], "y" : [ 2, 7] },
+                { "item" : "homeguns",   "chance" : 20, "x" : [ 2, 2], "y" : [10,11] },
+                { "item" : "dresser",    "chance" : 80, "x" : [ 2, 2], "y" : [10,11] },
+                { "item" : "dresser",    "chance" : 50, "x" : [ 2, 2], "y" : [ 7, 7] },
+                { "item" : "fridge",     "chance" : 50, "x" : [19,19], "y" : [11,11] },
+                { "item" : "kitchen",    "chance" : 50, "x" : [21,21], "y" : [11,11] },
```

Comment:
It was either this or converting y: [11, 11] to y: 11 in other places.
I chose this since it is easier to read imo.